style.css

```diff
@@ -34,31 +46,63 @@ body {
 }
 
 .pages a {
-  font-family: "Poppins", sans-serif;
+  font-family: Poppins, sans-serif;
   font-style: normal;
-  font-weight: 500;
-  font-size: 15px;
-  color: #344563;
+  color: white;
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 44px;
+  text-align: left;
+}
+
+.pages.active {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  background-color: rgba(96, 112, 255, 1);
+  mix-blend-mode: multiply;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  padding-top: 105px;
+  padding-left: 40px;
+  gap: 1.2rem;
+  z-index: 100;
+}
```

Comment:
[OPTIONAL]
- [ ] Nice work implementing the mobile menu, it looks really good :clap: However it would be nice to add `position: fixed` to the mobile menu to prevent it scrolling with the page.